consensus/src/quorum_store/batch_coordinator.rs

```diff
@@ -109,6 +109,7 @@ impl BatchCoordinator {
     async fn handle_batches_msg(&mut self, author: PeerId, batches: Vec<Batch>) {
         if let Err(e) = self.ensure_max_limits(&batches) {
             warn!("Batch from {}: {}", author, e);
```

Comment:
should this be error!(), and we get an alert? seems like this should never happen - unless there is an actual issue or malicious node - and that is generally what we are fine with having error on